remappers/kubernetesmetrics/clustermetrics.go

```diff
@@ -0,0 +1,52 @@
+package kubernetesmetrics
+
+import (
+	remappers "github.com/elastic/opentelemetry-lib/remappers/internal"
+
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/pmetric"
+)
+
+func addClusterMetrics(
+	src, out pmetric.MetricSlice,
+	_ pcommon.Resource,
+) error {
+	var timestamp pcommon.Timestamp
+	var node_allocatable_memory, node_allocatable_cpu int64
+
+	// iterate all metrics in the current scope and generate the additional Elastic kubernetes integration metrics
+	dataset := "kubernetes.node"
```

Comment:
Can we make those constants defined on top level?

Reply:
See answer below